index/scorch/persister.go

```diff
@@ -35,16 +37,27 @@ import (
 
 var DefaultChunkFactor uint32 = 1024
 
-// Arbitrary number, need to make it configurable.
-// Lower values like 10/making persister really slow
-// doesn't work well as it is creating more files to
-// persist for in next persist iteration and spikes the # FDs.
-// Ideal value should let persister also proceed at
-// an optimum pace so that the merger can skip
-// many intermediate snapshots.
-// This needs to be based on empirical data.
-// TODO - may need to revisit this approach/value.
-var epochDistance = uint64(5)
+var DefaultWorkLoopWait int = 2000 // ms
+
+var MaxWorkLoopWait int = 5000 // ms
```

Comment:
wondering why have a MaxWorkLoopWait?

Reply:
as it is configurable, thought of making a max check as well. 
But this may also end up being an advanced, rarely used configuration setting too.,